libwild/src/layout.rs

```diff
@@ -2384,6 +2412,18 @@ impl<'data> GroupState<'data> {
             .into_iter()
             .map(|file| file.finalise_layout(memory_offsets, resolutions_out, resources))
             .collect::<Result<Vec<_>>>()?;
+
+        if let Some(epilogue_layout) = files.iter().find_map(|file| {
+            if let FileLayout::Epilogue(epilogue) = file {
+                Some(epilogue)
+            } else {
+                None
+            }
+        }) {
+            let size = epilogue_layout.sysv_hash_size.unwrap_or(0);
+            *self.common.mem_sizes.get_mut(part_id::SYSV_HASH) = size;
```

Comment:
Could this and the computation of sysv_hash_size move to `EpilogueLayoutState::finalise_sizes`?